src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -224,3 +224,22 @@ def app_teardown(
     )
     processor.process(force)
     return MessageResult(f"Teardown is now complete.")
+
+
+@app.command("deploy", requires_connection=True)
+@with_project_definition("native_app")
+def app_deploy(
+    **options,
+) -> CommandResult:
+    """
+    Syncs the local changes to the stage without creating or updating the application.
+    """
```

Comment:
This needs to be more descriptive: 
Creates an application package in your Snowflake account and syncs the local changes to the stage without creating or updating the application.